core/modules/xrdsvc/SsiSession.cc

```diff
@@ -69,7 +69,7 @@ void SsiSession::ProcessRequest(XrdSsiRequest* req, unsigned short timeout) {
 
     auto replyChannel = std::make_shared<ReplyChannel>(*this);
 
-    auto errorFunc = [this, &req, &replyChannel](std::string const& errStr, bool sendStr=true ) {
+    auto errorFunc = [this, &req, &replyChannel](std::string const& errStr, bool sendStr) {
```

Comment:
I talked with Andy and there's no reason not to have an error message, so it's probably better to eliminate the sendStr argument and anywhere it is false, just make an appropriate error message. I think there's only one and there's already an error message.